overlord/devicestate/devicectx.go

```diff
@@ -104,6 +104,17 @@ func (dc groundDeviceContext) HasModeenv() bool {
 	return dc.model.Grade() != asserts.ModelGradeUnset
 }
 
+// IsCoreBoot is true when there are modes, or when there are not but
+// we are not in classic (UC16/18 case)
+func (d *groundDeviceContext) IsCoreBoot() bool {
```

Comment:
we don't have it but maybe it is time to start a devicectx_test.go ?

Reply:
Added now